specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+              }
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Retrieve Certificate Email History": {
+            "$ref": "./examples/RetrieveCertificateEmailHistory.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AppServiceCertificate": {
+      "description": "Key Vault container for a certificate that is purchased through Azure.",
+      "type": "object",
+      "properties": {
+        "keyVaultId": {
+          "description": "Key Vault resource Id.",
+          "type": "string"
+        },
+        "keyVaultSecretName": {
+          "description": "Key Vault secret name.",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Status of the Key Vault secret.",
+          "enum": [
+            "Initialized",
+            "WaitingOnCertificateOrder",
+            "Succeeded",
+            "CertificateOrderFailed",
+            "OperationNotPermittedOnKeyVault",
+            "AzureServiceUnauthorizedToAccessKeyVault",
+            "KeyVaultDoesNotExist",
+            "KeyVaultSecretDoesNotExist",
+            "UnknownError",
+            "ExternalPrivateKey",
+            "Unknown"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "KeyVaultSecretStatus",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "AppServiceCertificateCollection": {
+      "description": "Collection of certificate order certificates.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AppServiceCertificateResource"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AppServiceCertificateOrder": {
+      "description": "SSL certificate purchase order.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "AppServiceCertificateOrder resource specific properties",
+          "required": [
+            "productType"
+          ],
+          "type": "object",
+          "properties": {
+            "certificates": {
+              "description": "State of the Key Vault secret.",
+              "type": "object",
+              "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=15&length=20)

additionalProperties pattern is not allowed. You already have a schema, can you just $ref it here? The properties are already optional (by default), so any subset can be provided in the PATCH payload.